src/markdown_renderers.js

```diff
@@ -114,9 +114,9 @@ export const generateGitRepoUrlPrefix = (repoUrl, branchName, href) => {
  * @return {string}
 */
 export const customLinkRenderer = (href, text, repoUrl, branchName) => {
-    // Handle anchor links and local Apify links
+    // Handle anchor links, local Apify links, and mailto
     // Return Apify domain links without rel="nofollow" for SEO
-    if (href.startsWith('#') || href.includes('apify.com')) {
+    if (href.startsWith('#') || href.includes('apify.com') || href.startsWith('mailto')) {
```

Comment:
How about creating a regex for this? Already began to grow...
Thinking about something like `/^(mailto|tel|sms):.*$/` for identification that the href is this type of URI scheme so there won't be a need to change this again in the future.

Reply:
Yeah, good idea!